schemas/common_definitions.json

```diff
@@ -209,6 +214,46 @@
     "additionalProperties": false,
     "required": ["years"]
   },
+  "value_reference": {
+    "oneOf": [
+      {
+        "description": "A string or integer value",
+        "type": ["integer", "string"]
+      },
+      {
+        "description": "A reference to an answer",
+        "type": "object",
+        "properties": {
+          "identifier": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/identifier",
+            "description": "The id of an answer to reference"
+          },
+          "source": {
+            "type": "string",
+            "enum": ["answers"]
+          }
+        },
+        "required": ["identifier", "source"],
+        "additionalProperties": false
+      },
+      {
+        "description": "A reference to a piece of metadata",
+        "type": "object",
+        "properties": {
+          "identifier": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/metadata_identifier",
+            "description": "The id of the metadata to reference"
```

Comment:
`#metadata_identifier` is the same as `#identifier`, maybe the description of the identifier should be changed instead of the duplication?

Reply:
No it isn't. Metadata allows use of underscores.

Reply:
Oh ya ofc 😂